sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/transforms/Filter.java

```diff
@@ -163,6 +175,11 @@ public void processElement(ProcessContext c) {
           c.output(c.element());
         }
       }
+
+      @Override
+      public void populateDisplayData(DisplayData.Builder builder) {
+        Filter.populateDisplayData(builder, String.format("x ≤ %s", value));
```

Comment:
Note also this'll get serialized as JSON and passed into the service. I don't see any reason we shouldn't support Unicode..